elements/map/helpers.ts

```diff
@@ -237,10 +238,15 @@ export async function registerProjectionFromCode(code: string | number) {
  */
 export function registerProjection(
   name: string,
-  projection: string | proj4.ProjectionDefinition
+  projection: string | proj4.ProjectionDefinition,
+  extent?: number[]
```

Comment:
In other places of `eox-map`, we have defined the type for extent as follows:
```js
  extent?: import("ol/extent").Extent
```
This also fits better with the `setExtent` function which expects an `Extent`.

Reply:
Also updated extent type definition, thanks!